From a45a0beb1b4c6ec4c3879cff29f3c042d9e76cff Mon Sep 17 00:00:00 2001
From: Jan Janssen <jan-janssen@users.noreply.github.com>
Date: Wed, 15 Jun 2022 07:12:50 -0600
Subject: [PATCH] Depend on scikit-learn directly rather than sklearn

---
 setup.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/setup.py b/setup.py
index 6afac1eb..85388d3e 100644
--- a/setup.py
+++ b/setup.py
@@ -25,7 +25,7 @@
                 'ipywidgets>=5.2.2',
                 'ipython>=5.1.0,<6;python_version<"3.3"',  # IPython 6.0+ does not support Python < 3.5
                 'ipython>=6.0;python_version>="3.3"',  # Beginning with IPython 6.0, Python 3.3 and above is required.
-                'sklearn',
+                'scikit-learn',
                 'scipy'
                 ]
 
